jcl/src/jdk.management/share/classes/com/ibm/lang/management/internal/ExtendedMemoryMXBeanImpl.java

```diff
@@ -84,30 +86,37 @@ protected void startNotificationThread() {
 				return thread;
 			};
 
-			Thread notifier = AccessController.doPrivileged(createThread);
+			Thread notifier = java.security.AccessController.doPrivileged(createThread);
 			notifier.start();
 		}
 	}
 
 	/**
 	 * Deprecated. Use getTotalPhysicalMemorySize().
 	 */
-	/*[IF Sidecar19-SE]
+	/*[IF JAVA_SPEC_VERSION > 8]*/
 	@Deprecated(forRemoval = true, since = "1.8")
-	/*[ELSE]*/
+	@SuppressWarnings("deprecation")
+	/*[ELSE] JAVA_SPEC_VERSION > 8 */
 	@Deprecated
-	/*[ENDIF]*/
+	/*[ENDIF] JAVA_SPEC_VERSION > 8 */
 	public long getTotalPhysicalMemory() {
 		return osinstance.getTotalPhysicalMemorySize();
 	}
 
+	/*[IF JAVA_SPEC_VERSION > 8]*/
+	@SuppressWarnings("deprecation")
+	/*[ENDIF] JAVA_SPEC_VERSION > 8 */
 	public long getTotalPhysicalMemorySize() {
 		return osinstance.getTotalPhysicalMemorySize();
 	}
 
 	/**
 	 * TODO Was this intended to be exposed in com.ibm.lang.management.MemoryMXBean?
 	 */
+	/*[IF JAVA_SPEC_VERSION >= 14] */
+	@Deprecated(since = "14")
+	/*[ENDIF] JAVA_SPEC_VERSION >= 14 */
 	public long getUsedPhysicalMemory() {
 		return osinstance.getTotalPhysicalMemorySize() - osinstance.getFreePhysicalMemorySize();
```

Comment:
It can be rewritten to use osinstance.getTotalMemorySize() - osinstance.getFreeMemorySize().